CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.18.0] Q1 2024
+- added ability to ability to authenticate against the dev and prod backends
```

Comment:
i guess this is refined. from an enduser point of view dev vs prod backends are probably confusing.

Reply:
typo: `ability to ability`

Reply:
I would prefer to only have the optional option `--endpoint my.endpoint` and the prod endpoint as default.
reason: most of our customers will use the multi-tenant prod env. exceptions from tha rule are omnect developers using the dev env and enterprise customers with an own instance of omnect cloud.

further we discussed that there might be potential improvements possible in future to get rid of the param, e.g. download an omnect-cli.conf file in portal

Reply:
@mlilien can imho be resolved